src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -127,31 +129,31 @@ def load_arguments(self, _):
 
     with self.argument_context('appconfig kv show') as c:
         c.argument('key', help='Key to be showed.')
-        c.argument('label', help="If no label specified, show entry with null label. Does NOT support filters like other commands")
+        c.argument('label', help="If no label specified, show entry with null label. Does NOT support filters like list command")
```

Comment:
Can you refine your help message to something like "If no label specified, show entry with null label.  Wildcard pattern is NOT supported, which is different from `az appconfig kv list` command."? Feel free to change to whatever you like. It is just my suggestion.

Reply:
In our doc.microsoft.com website, we have an explanation about filtering such as "*" and ",". It is not a full regex support and we don't have concept of wildcard matching. So I would just say "Filtering is not supported"

---
In reply to: [347081296](https://github.com/Azure/azure-cli/pull/11036#discussion_r347081296) [](ancestors = 347081296)

Reply:
Please use active voice "Do" here and can you provide link for filters documentation?

Reply:
Here is filtering [doc](https://github.com/Azure/azure-cli/pull/11036/commits/9d16e2d0cdf764ea96df79fb28456a95dfeb5b4d) 
Also we already revised the help message to be "Filtering is not supported." Check the latest [commit](https://github.com/Azure/azure-cli/pull/11036/commits/9d16e2d0cdf764ea96df79fb28456a95dfeb5b4d)